mc2/controllers/docker/models.py

```diff
@@ -112,6 +114,23 @@ def get_marathon_app_data(self):
                         int(settings.MESOS_DEFAULT_TIMEOUT_SECONDS),
                 }]
             })
+        elif self.marathon_health_check_cmd:
+            app_data.update({
+                "ports": [0],
```

Comment:
Why are we setting ports here?

Reply:
🤔 artefact from other health check -- I'll remove too